opencti-platform/opencti-front/src/utils/filters/SearchEntitiesUtil.ts

```diff
@@ -18,15 +18,24 @@ export const getOptionsFromEntities = (
   if (isStixObjectTypes.includes(filterKey)) {
     if (searchScope[filterKey] && searchScope[filterKey].length > 0) {
       filteredOptions = (entities[filterKey] || [])
-        .filter((n) => searchScope[filterKey].some((s) => (n.parentTypes ?? []).concat(n.type).includes(s)))
-        .sort((a, b) => (b.type ? -b.type.localeCompare(a.type) : 0));
+        .filter((n) => searchScope[filterKey].some((s) => (n.parentTypes ?? []).concat(n.type).includes(s)));
     } else {
-      filteredOptions = (entities[filterKey] || []).sort((a, b) => (b.type ? -b.type.localeCompare(a.type) : 0));
+      filteredOptions = (entities[filterKey] || []);
     }
   } else if (entities[filterKey]) {
     filteredOptions = entities[filterKey];
   }
-  return filteredOptions;
+  return filteredOptions.map((f) => (f.group ? f : { ...f, group: f.type }))
```

Comment:
Here we map and sort every list.
So we do a mapping. Replace for the isStixObject case where `group ` is `undefined`.
Then we sort by group and in alphabetic order